Makefile

```diff
@@ -9,9 +9,9 @@ ifeq (test_one,$(firstword $(MAKECMDGOALS)))
 endif
 
 clean:
-	find -name '*.pyc' -delete
-	find -name '*~' -delete
-	find -name '__pycache__' -type d -delete
+	find . -name '*.pyc' -delete
+	find . -name '*~' -delete
+	find . -name '__pycache__' -type d -delete
```

Comment:
I see - does that mean that if we include this change, the rest can work natively on MacOS? I'm happy to include it regardless, but for the future I wonder if it's worth the effort to support multiple environments to run it, especially now that we have Vagrant integration.